uiowa/uiowa
# Update news and events styling issues for i4429, i4430, i4431.

Updated the styling issues for the news and events block on i4429, i4430, and i4431. There are still two issues that I have discussed with @bspeare .

Resolves #4429 
Resolves #4430 
Resolves #4431 
Resolves #4440

- [x] 1px extra space below card__label on events.
- [x] Events block on separate page did not update card__date to card__subtitle and containing parent div is missing card--list.

# How to test

No css styles were changed, so pull as normal as check out the differences:
Before:
![Screen Shot 2021-11-12 at 11 57 00 AM](https://user-images.githubusercontent.com/3067017/141513033-3bbc2c06-8a5c-4bdf-81cb-ff901d4eba94.png)

After:
![Screen Shot 2021-11-12 at 11 57 18 AM](https://user-images.githubusercontent.com/3067017/141513038-d033bbe5-b96d-413e-97bc-82a7a131766d.png)

Also, the no results display with fixed:
![Screen Shot 2021-11-12 at 12 03 21 PM](https://user-images.githubusercontent.com/3067017/141513829-02636ee4-f5ac-42c3-bbfd-fe141695643c.png)
